docs/content/preview/migrate/install-yb-voyager.md

```diff
@@ -60,41 +71,68 @@ The export directory has the following sub-directories and files:
 ## Install yb-voyager
 
 YugabyteDB Voyager consists of the yb-voyager command line executable. yb-voyager keeps all of its migration state, including exported schema and data, in a local directory called the [*export directory*](#create-an-export-directory).
-<!-- For more information, refer to [Export directory](../../yb-voyager/reference/#export-directory). -->
```

Comment:
This is misplaced. No need to talk about export-dir while installing voyager. I suggest to drop the entire paragraph.

Reply:
Thats what i always thought Thanks for the clarification.